### PR TITLE
minor de_de changes

### PR DESCRIPTION
**Pull request by t-brieger**

### CODE REVIEW

**Comment by nift4** [comment]
@AlexIIL These changes are good picks.

File: assets/buildcraft/lang/de_DE.lang [hunk 1]
```diff
@@ -98,10 +98,10 @@ config.blueprints.excludedBlocks=Ausgeschlossene Blöcke
 config.builders.dropBrokenBlocks=Abgebaute Blöcke nicht zerstören
 
 config.display.colorBlindMode=Farbenblind-Modus
-config.display.hideFluidValues=Flüssigkeitszahlen verstecken
-config.display.hidePowerValues=Energiezahlen verstecken
+config.display.hideFluidValues=Flüssigkeitswerte verstecken
+config.display.hidePowerValues=Energiewerte verstecken
 
-config.experimental.kinesisPowerLossOnTravel=Energieleitungs-Energieverlust
+config.experimental.kinesisPowerLossOnTravel=Energieleitungs-verlust
```
Comment (Vexatos) [inline assets/buildcraft/lang/de_DE.lang]:
```suggestion
config.experimental.kinesisPowerLossOnTravel=Energieleitungs-Verlust
```

**Comment by nift4** [comment]
@Vexatos +1

**Comment by nift4** [comment]
@ichbineinNerd

**Pull request merged**